dynawo/sources/Modeler/DataInterface/PowSyblIIDM/DYNLoadInterfaceIIDM.h

```diff
@@ -143,6 +143,12 @@ class LoadInterfaceIIDM : public LoadInterface, public InjectorInterfaceIIDM {
     country_ = country;
   }
 
+#if _DEBUG_
+  powsybl::iidm::Load& getUnderlyingLoad() {
+    return loadIIDM_;
```

Comment:
Discussion ongoing with powsybl team on this one

Reply:
Removed